src/aliceVision/image/io.cpp

```diff
@@ -90,7 +90,12 @@ bool isSupported(const std::string &ext)
     for(std::string& extension: extensions)
       supportedExtensions.push_back(extension.insert(0, "."));
   }
+  return supportedExtensions;
+}
 
+bool isSupported(const std::string& ext)
+{
+  const std::vector<std::string> supportedExtensions = getSupportedExtensions();
```

Comment:
We could add static here:
`static const std::vector<std::string> supportedExtensions = getSupportedExtensions();`

Reply:
Fixed in https://github.com/alicevision/AliceVision/pull/806/commits/f7584167ca69f68bb430cb92284b5733df155fd7 .